src/main/java/cz/xtf/openshift/OpenShiftWaiters.java

```diff
@@ -190,4 +222,23 @@ public Waiter areExactlyNPodsRunning(int n, String key, String value) {
 	private Waiter areExactlyNPodsRunning(int n, Supplier<List<Pod>> podSupplier) {
 		return new SupplierWaiter<>(podSupplier, ResourceFunctions.areExactlyNPodsRunning(n), TimeUnit.MINUTES, 3);
 	}
+
+	/**
+	 * Creates a waiter that waits until there aren't any pods in project.
+	 * Defaults to 3 minutes timeout.
+	 *
+	 * @param key label key for pod filtering
+	 * @param value label value for pod filtering
+	 * @return Waiter instance
+	 */
+	public static Waiter areNoPodsPresent(String key, String value) {
```

Comment:
Isn't following use case sufficient for this purpose?

`areExactlyNPodsRunning(0, key, value)`

Reply:
Unfortunately not, need to be sure there aren't any pods. Your proposed solution, doesn't cover other Pod's state (Pending, Failed,...)

Reply:
Rethinking that and you are correct. It would actually work if you look at the implementation, but looking at the implementation, it's not exactly correct. Also, yep, from method name you could expect different behavior.